### ISSUE TITLE
No way to build

### ISSUE THREAD

**Issue by haf** [xyncro/chiron#84]
There's no `build.sh` or equivalent in the repository. Running `xbuild Chiron.sln` results in:

```
Chiron.fs(6,6): error FS0039: The namespace or module 'Aether' is not defined.
```

and similar.

**Comment by neoeinstein** [comment]
Are you targeting .NET Portable? This is likely my error in updating Aether. I will unlist that package in the meantime.

**Comment by haf** [comment]
No, it's not the package itself, it's the repo.

**Comment by neoeinstein** [comment]
Ah, you mean no way to build on *nix, rather than on Windows?

**Comment by haf** [comment]
Are there mono debug symbols published for the latest release that I can drop in?

**Comment by neoeinstein** [comment]
No. I only see the default pdbs in the published package. You should be able to drop in the default ProjectScaffold `build.sh`, with the modification that `build.fsx` is actually `paket-files\build\xyncro\build\build.fsx`. I will see if I can use that to get it building tonight.

**Comment by haf** [comment]
I tried that, it still doesn't compile.

**Comment by neoeinstein** [comment]
Ok. Will look into this tonight.

**Comment by haf** [comment]
Thanks @neoeinstein
